play-v27/src/main/scala/com/gu/googleauth/auth.scala

```diff
@@ -21,6 +21,7 @@ import play.api.mvc.{RequestHeader, Result}
 import scala.concurrent.{ExecutionContext, Future}
 import scala.language.postfixOps
 import scala.util.{Failure, Success, Try}
+import play.api.libs.ws.WSBodyWritables._
```

Comment:
Without this extra import we get an [`E172`](https://github.com/lampepfl/dotty/blob/6b12bf024b7b109885e8bb021fb9bf9cbb90cebe/compiler/src/dotty/tools/dotc/reporting/ErrorMessageID.scala#L188) error, reporting missing implicit arguments:

```
[error] -- [E172] Type Error: /Users/Roberto_Tyley/code/play-googleauth/play-v30/src/main/scala/com/gu/googleauth/auth.scala:229:7 
[error] 229 |      }.flatMap { response =>
[error]     |       ^
[error]     |Cannot find an instance of Map[K, Seq[String]] to WSBody. Define a BodyWritable[Map[K, Seq[String]]] or extend play.api.libs.ws.ahc.DefaultBodyWritables
[error]     |
[error]     |where:    K is a type variable with constraint >: String
[error]     |
[error]     |
[error]     |One of the following imports might fix the problem:
[error]     |
[error]     |  import play.api.libs.ws.DefaultBodyWritables.writeableOf_urlEncodedForm
[error]     |  import play.api.libs.ws.WSBodyWritables.writeableOf_urlEncodedForm
[error]     |  import play.api.libs.ws.writeableOf_urlEncodedForm
[error]     |
```

It's kind of interesting that although this error is about _missing implicit arguments_, and [it's called that in the Scala compiler code](https://github.com/lampepfl/dotty/blob/6b12bf024b7b109885e8bb021fb9bf9cbb90cebe/compiler/src/dotty/tools/dotc/reporting/ErrorMessageID.scala#L188), the error message displayed to the user doesn't actually use those words - it says 'Type Error', and recommends imports (👍 ) but doesn't use the words 'missing' or 'implicit'/['given'](https://docs.scala-lang.org/scala3/reference/contextual/givens.html).